### DISCUSSION TITLE
Hint on the offset selector is incorrect

### DISCUSSION THREAD

**Discussion by Banana090** [ppy/osu#30936] [Q&A] [answered]
I'm pretty sure the hints "hit objects appear **earlier/later**" is swapped
![image](https://github.com/user-attachments/assets/847ee9a3-5b20-4874-9dc4-b202de4a3689)

Posting in the discussion instead of issues since I'm not 100% sure, but around 95% I am.
Checked the music's waveform and offset, figured out I need objects to appear 20ms later, so offset would be +20, but the hint shows it "would appear earlier", tried doing -20 instead, didn't hit anything, changed to +20 -> all good

**Comment by peppy** [comment, marked as answer]
Offsets are hard, maybe your perspective is back to front.

The wording here is correct for the majority, though.

You can test by pausing autoplay and adjusting to see the results in realtime:

https://github.com/user-attachments/assets/9dbfc4bc-bd77-425a-ba9b-5853923e1663
